### ISSUE TITLE
[Submission] Texture, localisation, recipe fix using CraftTweaker

### ISSUE THREAD

**Issue by drakray** [DoubleDoorDevelopment/Lumberjack#25]
[Lumberjacks.zip](https://github.com/DoubleDoorDevelopment/Lumberjack/files/2800120/Lumberjacks.zip)
https://pastebin.com/Y9NEivbd

Texture, JSON & localisation in the ZIP.
Missing recipe and 2 fix in the Pastebin(as seen in my pack with Base/Modern Metals, Aquaculture, Abyssalcraft, NetherEx, Gravestone-Extended, Twilight Forest & Botania)

All Special Texture based on their respective mod's Axe, using Paint.NET
All Texture using color palette from corresponding mod's Axe, using Paint.NET

**Comment by Claycorp** [comment]
I am unsure what this is for...
Are the textures you provide missing from the base mod and need to be added?
What version is this for?
What is the script for?

Please explain, thanks.

**Comment by drakray** [comment]
Sorry for that, Yes, missing texture/json, for Lumberjack 1.12.2-1.3.1

The script is a CraftTweaker one, to add recipe for LumberAxe, because I don't know why but only vanilla and AbyssalCraft-based LumberAxe have recipe(and Ethaxium lumberaxe don't use EthaxiumBrick as handle, like all AC Ethaxium tools, fixed in script)
Script is for anyone to use/tweak/adapt to their need, instead of having to start one from scratch, if they want to

**Comment by Claycorp** [comment]
Oh so some axes DO work.
The 1.12 port wasn't done by me and I tested a few mods and nothing would come up. It is on my list of to look into.
As for the recipes, that would be due to people not adding repair materials IIRC. Recipes are always generated with sticks as there is no way to tell what it should be otherwise.

I'll get these textures added sometime soon and see if I can get any other axes to show up that should. Thanks for them!

**Comment by drakray** [comment]
My pleasure :)
another thing, adding materials in materials.json don't automatically add recipes either
[JSON in config/d3core](https://pastebin.com/vj7aQPjA)

**Comment by Claycorp** [comment]
D3Core is slowly getting phased out as library mod. I don't care for "library" mods myself and it makes it easier for people to take over if anything happens to D3. I can look into it but as Lumberjack still needs D3Core stripped it may take a bit to replace it all.

Also the material issue should be going away in 1.13 as they have added a repair stack when you define the material. Like it should have been forever ago....

**Comment by drakray** [comment]
:+1: 
I just hope that doesn't mean 1.12.2 version will stop getting love ;) :P

**Comment by Claycorp** [comment]
So I was poking around some at this... 

- I converted it away from D3Core so you no longer need D3Core for it.
- I looked into the materials issue and that method isn't going to work anymore without changing how the recipes are done. I don't know if it's even possible to fix at this moment.
- The deal with some mods adding axes and some not (thermal for example) isn't totally known to me. I have a feeling it's because Thermal is registering after Lumberjack is. I know of one fix but it's not really a good way to go about it....
- I have yet to merge in your textures though, I will do that next and get a new build up with the new textures.

Just thought I would get you an update on all of this.
